Lib/test/test_pdb.py

```diff
@@ -3935,6 +3935,25 @@ def test_readrc_homedir(self):
                     f.write("invalid")
                 self.assertEqual(pdb.Pdb().rcLines[0], "invalid")
 
+    def test_readrc_current_dir(self):
+        with os_helper.temp_cwd() as cwd:
+            rc_path = os.path.join(cwd, ".pdbrc")
+            with open(rc_path, "w") as f:
+                f.write("invalid")
+            self.assertEqual(pdb.Pdb().rcLines[0], "invalid")
```

Comment:
I think you should combine this line and the line below. Just check `pdb.Pdb().rcLines` directly.

Reply:
I've changed it `self.assertEqual(pdb.Pdb().rcLines[-1], "invalid")`, this test is not patching a `.pdbrc` at `$HOME` so potentially there can be other lines first in rcLines, so now it only checks that the last one comes from the file written during the test


incidentally, i just found there are 5 other tests that fail if you do happen to have a non-empty .pdbrc at $home, it's ok in CI because there is none, but locally a few unrelated things fail when the file is present